src/hooks/use-format-localized-timestamps.tsx

```diff
@@ -0,0 +1,41 @@
+import { Duration, intervalToDuration } from 'date-fns';
+import { useCallback } from 'react';
+import { formatDistance } from '../lib/date';
+import { SupportedLocale } from '../lib/locale';
+import { useI18nData } from './use-i18n-data';
+
+function formatTimeDistance( duration: Duration, _locale: SupportedLocale ): string {
```

Comment:
Let's remove this `_locale` variable and all its dependencies like `useI18nData`.

```suggestion
function formatTimeDistance( duration: Duration ): string {
```

Reply:
I did some cleanup here: https://github.com/Automattic/studio/pull/714/commits/201425f7820cd8c02caf2d2b31720df1623dd208